### DISCUSSION TITLE
some idea about ux

### DISCUSSION THREAD

**Discussion by todpeng777** [anapnoe/stable-diffusion-webui-ux#14] [Ideas]
Thank you so much for such a great product, it's awesome!
But I still feel some inconvenience during the use process, especially the prompt part, so I have some ideas about ue,about the prompt, can we make them into tabs, so that we can easily move the position, group, and use buttons to operate their weights,,,
And about several important panels, I drew a sketch to show my idea
![uedidea](https://user-images.githubusercontent.com/125846611/226643201-8ab3688d-b5ba-46f5-8dcd-b92bb0759ef9.png)

**Comment by anapnoe** [comment]
can you create the mockup on the actual ui/ux it would be much more helpful to visualize and one Idea each mockup then you can add them on issues as [enhancement] I like them thank you

**Reply by todpeng777** [reply]
I am sorry I don't know how to create an mockup on the actual ux,,,,I  can only use photoshop，，，，
